cmd/cli/uninstall_mesh.go

```diff
@@ -141,6 +141,24 @@ func (d *uninstallMeshCmd) run() error {
 		}
 
 		if err == nil {
+			namespaces, _ := selectNamespacesMonitoredByMesh(d.meshName, d.clientSet)
+			if len(namespaces.Items) != 0 {
```

Comment:
@mudit-01 I don't think this check is necessary - we could remove one level of indentation. `range` is expected to handle nils (valid empty list) like this: https://go.dev/play/p/J7JQ8LKVYi6

Reply:
Thanks, @draychev for the review 👍🏽